lib/ansible/playbook/block.py

```diff
@@ -298,8 +298,10 @@ def _get_parent_attribute(self, attr, omit=False):
         '''
         Generic logic to get the attribute or parent attribute for a block value.
         '''
-        extend = self.fattributes.get(attr).extend
-        prepend = self.fattributes.get(attr).prepend
+        fattr = self.fattributes[attr]
+
+        extend = getattr(fattr, 'extend', False)
+        prepend = getattr(fattr, 'prepend', False)
```

Comment:
dont need getattr here, extend/prepend are always part of FAs

Reply:
They are not part of `NonInheritableFieldAttribute` but that _should not_ happen in `_get_parent_attribute` so yes `getattr` is likely not needed.

Reply:
I'm not going to make that change now.  We can do it for 2.15.  I questioned making this change altogether to limit the impact.

Reply:
actually, the old code didn't care, I'll just make it now.